### DISCUSSION TITLE
How to import KEDA v2.0.0-beta

### DISCUSSION THREAD

**Discussion by zzxwill** [kedacore/keda#1198] [Q&A / Need Help] [answered]
I'd like to use KEDA client go, so I tried to import KEDA v2.0.0-beta.
```
import github.com/kedacore/keda v2.0.0-beta
```
But got the following issue.
```
/Users/zhouzhengxi/Programming/golang/src/github.com/zzxwill/oam-autoscaler-trait/go.mod:8: require github.com/kedacore/keda: version "v2.0.0-beta" invalid: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v2
```

Question: How to import KEDA v2?

**Comment by zroubalik** [comment, marked as answer]
go has some restrictions and limitations on importing modules and theirs versions (imho it doesn't like the beta in there).

So I'd recommend you to point to the specific commit hash you I'd like to import:

eg.
```
import github.com/kedacore/keda COMMIT_HASH
```

and the run `go mod tidy` to resolve that particular commit.
